packages/backend/src/nest/libp2p/process-in-chunks.service.ts

```diff
@@ -2,7 +2,7 @@ import { EventEmitter } from 'events'
 import fastq, { queueAsPromised } from 'fastq'
 
 import Logger from '../common/logger'
-import { randomUUID } from 'crypto'
+import CryptoJS from 'crypto-js'
```

Comment:
This says it's deprecated and not maintained. I think that's a barrier to us using it unless there's something I'm missing? https://www.npmjs.com/package/crypto-js

Reply:
It was already in the project and I'm only using it to generate a consistent shared ID via hashing, no security threat or anything.

Reply:
And in the other place it's not used for anything security related either?

Reply:
That I don't know, I haven't checked the usages